File: mobile-android/app/src/main/java/com/example/p2paudio/audio/PcmFrame.kt
package com.example.p2paudio.audio

data class PcmFrame(
    val sequence: Int,
    val timestampMs: Long,
    val sampleRate: Int,
    val channels: Int,
    val bitsPerSample: Int,
    val frameSamplesPerChannel: Int,
    val pcmBytes: ByteArray
)

File: desktop-windows/src/P2PAudio.Windows.Core/Audio/PcmFrame.cs
namespace P2PAudio.Windows.Core.Audio;

public sealed record PcmFrame(
    int Sequence,
    long TimestampMs,
    int SampleRate,
    int Channels,
    int BitsPerSample,
    int FrameSamplesPerChannel,
    byte[] PcmBytes
);
